config/custom.css

```diff
@@ -1,738 +1,376 @@
-body {
-    background: url(http://i.imgur.com/SrWXesn.jpg?1) no-repeat;
-    background-color: #999CA1;
-}
-
-#header {
-    background: url(http://i.imgur.com/XkjakIM.png)
-}
-
-.ps-room.ps-room-light.scrollable {
-    color: #FFF;
-    background: rgba(44,44,44,0.8);
-}
-
-.menugroup {
-    background: rgba(255,255,255,0)
-}
-
-.battleform label {
-    background: url(http://i.imgur.com/yFWeHx0.png) no-repeat scroll left center transparent!important;
+@-moz-document domain("play.pokemonshowdown.com") {
```

Comment:
This line breaks the entire CSS.
Remove it (and the indentation it causes to the entire file)